charts/kyverno/values.yaml

```diff
@@ -160,7 +160,7 @@ config:
   - "[TokenReview,*,*]"
   - "[SubjectAccessReview,*,*]"
   - "[SelfSubjectAccessReview,*,*]"
-  - "[*,kyverno,*]"
+  - "[*,kyverno,kyverno*]"
```

Comment:
Looks like the namespace and the name can be configured via Helm chart:
https://github.com/kyverno/kyverno/blob/25c2ad11e4c62f36796c390abb1b3b71d4f71e26/charts/kyverno/values.yaml#L1-L3

How do handle them dynamically?